src/components/attachments-list/attachments-list.tsx

```diff
@@ -12,16 +12,18 @@ interface AttachmentsListProps {
   attachments: Array<KalturaAttachmentAsset>;
   downloadPluginManager: DownloadPluginManager;
   attachmentsLabel?: string;
+  downloadAttachmentsButtonLabel?: string;
```

Comment:
move common stuff that includes - downloadPluginManager & downloadAttachmentsButtonLabel

Reply:
done